### PR TITLE
first draft of changes for XDB-1052

### PR DESCRIPTION
**Pull request by drothery-edb**
## What Changed?

- Updated RPM install topic
- Changed title of RPM upgrade topic in the appendix

### CODE REVIEW

**Review by nidhibhammar — APPROVED** [review]

File: product_docs/docs/eprs/7.0/03_installation/03_installing_rpm_package.mdx [hunk 1]
```diff
@@ -6,17 +6,9 @@ title: "Installing the Replication Server RPM Package"
...
-> <http://yum.baseurl.org/>
-
-To request credentials to the EDB Yum Repository, visit the following website:
-
-> <https://www.enterprisedb.com/repository-access-request>
-
-For information about using the EDB Yum Repository see Chapter 3 of the EDB Postgres Advanced Server Installation Guide available from the EnterpriseDB website located at:
-
-> <https://www.enterprisedb.com/resources/product-documentation>
+For information about using the EDB yum repository see the [EDB Postgres Advanced Server Linux Installation Guide](/epas/latest/epas_inst_linux).
```
Comment (ebgitelman) [inline product_docs/docs/eprs/7.0/03_installation/03_installing_rpm_package.mdx:11]:
```suggestion
For information about using the EDB yum repository, see the [EDB Postgres Advanced Server Linux Installation Guide](/epas/latest/epas_inst_linux).
```

File: product_docs/docs/eprs/7.0/03_installation/03_installing_rpm_package.mdx [hunk 2]
```diff
@@ -260,8 +254,43 @@ Successful installation of Replication Server results in the creation of directo
...
 `yum` will update the `edb.repo` file to enable access to the current EDB repository, configured to connect with the credentials specified in your `edb.repo` file. Then, you can use yum to upgrade any installed packages:
 
-> `yum upgrade ppas-xdb*`
+`yum upgrade ppas-xdb*`
+
+See [Upgrading from a Replication Server RPM Package Installation](../10_appendix/02_upgrading_to_xdb6_2/03_upgrading_with_xdb_rpm_package/) for details.
+
+## Updating a Bitrock Installer Installation
+
+If you are upgrading from a 6.2.x version of Replication Server that was installed using the Replication Server Bitrock installer, perform the following steps:
```
Comment (ebgitelman) [inline product_docs/docs/eprs/7.0/03_installation/03_installing_rpm_package.mdx:267]: You can. upgrade from a 6.2.x version of Replication Server that was installed using the Replication Server Bitrock installer.

File: product_docs/docs/eprs/7.0/03_installation/03_installing_rpm_package.mdx [hunk 3]
```diff
@@ -260,8 +254,43 @@ Successful installation of Replication Server results in the creation of directo
...
+
+1. Stop the Replication Server Publication and Subscription services.
+
+1. Take a backup of the following file and folder:
```
Comment (ebgitelman) [inline product_docs/docs/eprs/7.0/03_installation/03_installing_rpm_package.mdx:271]: Back up the following file and folder:

File: product_docs/docs/eprs/7.0/03_installation/03_installing_rpm_package.mdx [hunk 4]
```diff
@@ -260,8 +254,43 @@ Successful installation of Replication Server results in the creation of directo
...
+   - `/opt/PostgreSQL/EnterpriseDB-xDBReplicationServer/xdata`
+
+1. If you have modified any of the following configuration files in the `/opt/PostgreSQL/EnterpriseDB-xDBReplicationServer/etc` folder, take a backup of the modified files:
+   - `encoding_map.conf`
+   - `euc_jp.conf` 
+   - `sysconfig` 
+   - `xdb_pubserver.conf` 
+   - `xdb_subserver.conf`
+
+1. Uninstall Replication Server 6.2.x using the following command:
```
Comment (ebgitelman) [inline product_docs/docs/eprs/7.0/03_installation/03_installing_rpm_package.mdx:282]: 1. Uninstall Replication Server 6.2.x:

File: product_docs/docs/eprs/7.0/03_installation/03_installing_rpm_package.mdx [hunk 5]
```diff
@@ -260,8 +254,43 @@ Successful installation of Replication Server results in the creation of directo
...
+
+   `/opt/PostgreSQL/EnterpriseDB-xDBReplicationServer/uninstall-xdbreplicationserver`
+
+1. Complete the [installation steps](#installation-steps) in this topic.
+
+6. After you have completed the installation steps, stop any running Publication and Subscription services.
```
Comment (ebgitelman) [inline product_docs/docs/eprs/7.0/03_installation/03_installing_rpm_package.mdx:288]: 6. After you complete the installation steps, stop any running Publication and Subscription services.

**Pull request merged**